dts/bindings/espi/microchip,xec-espi-saf.yaml

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2019 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+description: Microchip ESPI SAF controller
+
+# Copyright (c) 2020 Microchip Technology Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+description: Microchip ESPI SAF controller
```

Comment:
Some copy/paste errors happened here.

Reply:
Sorry I was not precise enough. The description has been fixed, but the SPDX line is still duplicated and the copyrights are not all together.

The top of this file should look like this:

```yaml
# Copyright (c) 2019 Intel Corporation
# Copyright (c) 2020 Microchip Technology Inc.
# SPDX-License-Identifier: Apache-2.0

description: Microchip ESPI SAF controller

compatible: "microchip,xec-espi-saf"
```

Reply:
Hi, fixed it.